pkg/admin/debug/debug_api.go

```diff
@@ -1,4 +1,4 @@
-// +build mosn_debug
+//go:build mosn_debug
```

Comment:
keep both build constraints, otherwise go<1.17 may fails.
ref: 
- https://github.com/golang/sys/blob/master/cpu/cpu_gccgo_x86.go#L5
- https://stackoverflow.com/questions/68360688/whats-the-difference-between-gobuild-and-build-directives

Reply:
gotcha I didn't realize go 1.18 also meant keep go 1.17 (which is no longer supported)

I will revert the changes. Please note that code using 1.18 `any` will also fail on 1.17